share/pvss/pvss.go

```diff
@@ -121,19 +135,19 @@ func DecShare(suite Suite, H kyber.Point, X kyber.Point, sH kyber.Point, x kyber
 	G := suite.Point().Base()
 	V := suite.Point().Mul(suite.Scalar().Inv(x), encShare.S.V) // decryption: x^{-1} * (xS)
 	ps := &share.PubShare{I: encShare.S.I, V: V}
-	P, _, _, err := dleq.NewDLEQProof(suite, G, V, x)
+	P, com, _, err := dleq.NewDLEQProof(suite, G, V, x)
```

Comment:
Same remark as `EncShares` here - we don't need to output `com`. To verify the proof of a `decShare`, we need (G, X[i], V, encShares[i]) <-> $(G, y_i, S_i, Y_i)$. Three parameters are already public, $(G, y_i, Y_i)$, and $S_i$ is output by `DecShare` alongside the proof.

Reply:
Fixed !